mysql-test/main/subselect_nulls_innodb.result

```diff
@@ -0,0 +1,26 @@
+#
+# MDEV-32090 Index does not handle null-safe equals operator correctly in join
+#
+CREATE TEMPORARY TABLE `Building` (
+`id` int(10) unsigned NOT NULL,
```

Comment:
Please use table names `t1`, `t2` and so forth.
Please also drop the table at testcase end.
Rationale: 
 * all other tests do so, it's better to follow their path.
 * temporary tables are dropped when server restarts, which makes it hard to debug the SELECT query if it crashes the server (you need to re-issue the CREATE TABLE after the server is restarted)
 * It's not a good practice for each testcase to leave behind its tables (which are not guaranteed to be unique)